test/Datadog.Trace.ClrProfiler.IntegrationTests/TestHelper.cs

```diff
@@ -264,9 +272,38 @@ protected void ValidateSpans<T>(IEnumerable<MockTracerAgent.Span> spans, Func<Mo
             }
         }
 
-        public class IISExpress : IDisposable
+        protected async Task AssertHttpSpan(
+            string path,
+            int agentPort,
+            int httpPort,
+            HttpStatusCode expectedHttpStatusCode,
+            string expectedSpanType,
+            string expectedOperationName,
+            string expectedResourceName)
+        {
+            List<MockTracerAgent.Span> spans;
+
+            using (var agent = new MockTracerAgent(agentPort))
+            using (var httpClient = new HttpClient())
+            {
+                var response = await httpClient.GetAsync($"http://localhost:{httpPort}" + path);
+                var content = await response.Content.ReadAsStringAsync();
+                Output.WriteLine($"[http] {response.StatusCode} {content}");
```

Comment:
Is this a forgotten debug line?

Reply:
It was [already there in the tests](https://github.com/DataDog/dd-trace-csharp/pull/217/files/ecd0d06ad9d02301e2285ebce5d326ef794bf6bb#diff-b20a667425a10cb7766d73ec0ececc21L56) as Caleb wrote them. The output shows up in the CI logs which can be helpful when the test fails.

Reply:
That seems very fragile to me, if I'm not mistaken XUnit runs the tests concurrently and I would expect the output to get all mixed up between the concurrent tests running, or even simply discarded by Xunit.
Xunit has a dedicated pattern to do this properly: https://xunit.github.io/docs/capturing-output

Reply:
That's the pattern we are following 😉 `Output` is a `ITestOutputHelper` that is injected through the test class's [constructor](https://github.com/DataDog/dd-trace-csharp/pull/217/files/ecd0d06ad9d02301e2285ebce5d326ef794bf6bb#diff-e43db7e43b76b3c566ace139fed6bcf5R18).

Reply:
Haha 😆Never mind, I'm just stupid.